README.md

```diff
@@ -1,5 +1,6 @@
 # HybridQ: A Hybrid Simulator for Quantum Circuits
 
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/nasa/hybridq/main)
```

Comment:
This link doesn't work for me. I get this error:
`
pull access denied for hybridq-baseline, repository does not exist or may require 'docker login': denied: requested access to the resource is deniedBuilt image, launching...
Failed to connect to event stream
`

Reply:
The link doesn't work yet as it points to the `main` branch. Use https://mybinder.org/v2/gh/nasa/hybridq/mybinder for testing.